lib/tlsca/ca.go

```diff
@@ -538,6 +538,10 @@ var (
 	// BotInstanceASN1ExtensionOID is an extension that encodes a unique bot
 	// instance identifier into a certificate.
 	BotInstanceASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 2, 20}
+
+	// UserTypeASN1ExtensionOID is an extension that encodes the user type.
+	// Its value is either local or sso.
+	UserTypeASN1ExtensionOID = asn1.ObjectIdentifier{1, 3, 9999, 2, 21}
```

Comment:
It looks like we were originally trying to namespace these extensions a bit better.

We had a bunch of attributes under 1.3.999.1.X, and then it looks like we incremented to 1.3.999.2.X for database-specific extensions, but then devs just started adding everything to the .2 section.

Just an observation, I'm not sure if you need to change anything.

Reply:
Updated to use `1.3.999.1.x` sequence